static/lang/en.json

```diff
@@ -2679,6 +2679,17 @@
                 "Hint": "Settings to exclude packs from loading",
                 "Name": "Compendium Browser Packs"
             },
+            "CompendiumBrowserSources": {
+                "DeleteAllInfo": "This can not be undone, but you may reconfigure this. The settings will be set to allow all sources, which include known sources, empty sources and unknown sources.",
+                "DeleteAllQuestion": "Are you sure you want to delete the filter sources settings?",
+                "DeleteAllTitle": "Delete filter sources settings",
+                "Hint": "Settings to display only entries with specified sources in the compendium browser.",
+                "IgnoreAsGM": "Let GMs ignore filtering and see all entries.",
+                "Name": "Compendium Browser Specify Sources",
+                "ShowEmptySources": "Do not filter entries with empty sources.",
+                "ShowUnknownSources": "Do not filter entries with unknown sources.",
```

Comment:
What is an unknown source?

Reply:
It is a source, which was not yet in the system when the settings where created, so it is missing in the list of sources. For Example:

- There are two sources s1 and s2 in the system at the moment. 
- We set it up that s1 is loaded and s2 is not loaded. 
- In an update s3 is introduced to the system.
 
In this example s3 is an unknown source, we do not have yet made a decision if s3 specifically should be loaded or not.

Maybe new sources would be better?